app.yaml

```diff
@@ -87,13 +87,13 @@ handlers:
   secure: always
   http_headers:
     Cache-Control: 'no-cache'
-# Serve js scripts for interactions, rich_text_components and objects under
-# extensions in dev mode. This regex allows us to recursively serve js scripts
-# under the three specified directories. "\1" and "\2" insert capture groups
-# from the url pattern.
-- url: /extensions/(interactions|rich_text_components|objects)/(.*\.(js))$
+# Serve js scripts for gadgets, interactions, rich_text_components and objects
```

Comment:
I thought I just got rid of gadgets in a recent PR? Seems like there's a merge error here.

Reply:
OK, my bad. Actually I did remove gadgets from code part but kept the comment from my version of file. So my version still had gadgets written in comment but it should not be present in code. I'll fix this comment. Thanks.